### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@webcomponents, anybody please merge this pull request.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@webcomponents/template",
   "version": "0.0.1",
-  "description": "Minimal template polyfills needed by webcomponents.js",
+  "description": "HTML Template polyfill",
```
Comment (notwaldorf) [inline package.json]: Why the change? The original description says exactly what this does

**Review by notwaldorf — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (notwaldorf) [reply package.json]: Also needs a rebase

File: package.json [hunk 2]
```diff
@@ -10,15 +10,25 @@
     "type": "git",
     "url": "https://github.com/webcomponents/template.git"
   },
-  "author": "The Polymer Authors",
+  "author": "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)",
```
Comment (notwaldorf) [inline package.json]: Also, as mentioned in many of the other PRs, pls don't make this change

File: package.json (hunk 1 above)
Comment (FluorescentHallucinogen) [reply package.json]: Is it possible to use this project as separate HTML Template polyfill without webcomponents.js? If yes, what difference does it make which project uses it?

File: package.json (hunk 2 above)
Comment (FluorescentHallucinogen) [reply package.json]: I agree with changing "Polymer Project Authors" to "Polymer Authors", but what's wrong with using URL to authors list in `bower.json`/`package.json`?

@azakus, WDYT?

**Comment by FluorescentHallucinogen** [comment]
All requested changes done.
@notwaldorf, @azakus, @bicknellr, @sorvell, @justinfagnani, please merge this pull request.

**Comment by FluorescentHallucinogen** [comment]
@notwaldorf, @azakus, @bicknellr, @sorvell, @justinfagnani PTAL.

**Comment by notwaldorf** [comment]
Closing, since this is a very old PR and needs a rebase

**Pull request closed without merging**